.changeset/short-cars-scream.md

```diff
@@ -0,0 +1,8 @@
+---
+'@firebase/analytics': patch
+'@firebase/util': patch
```

Comment:
`util` should be minor. Can you also add what new functions are added to `util`?

Reply:
Done, I suppose firebase should be minor as well then?

Reply:
no, because it doesn't cause any API change in `firebase`